docs/user-guide/min-max-timeslice-constraints.ipynb

```diff
@@ -30,28 +30,28 @@
     "\n",
     "We will modify the minimum service factor for `gasCCGT` in the `power` sector as follows.\n",
     "\n",
-    "|ProcessName|RegionName|Time|month|day|hour|UtilizationFactor|MinimumServiceFactor|\n",
-    "|-|-|-|-|-|-|-|-|\n",
-    "|Unit|-|Year|-|-|-|-|-|\n",
-    "|gasCCGT|R1|2020|all-year|all-week|night|1|**0.2**|\n",
-    "|gasCCGT|R1|2020|all-year|all-week|morning|1|**0.4**|\n",
-    "|gasCCGT|R1|2020|all-year|all-week|afternoon|1|**0.6**|\n",
-    "|gasCCGT|R1|2020|all-year|all-week|early-peak|1|**0.4**|\n",
-    "|gasCCGT|R1|2020|all-year|all-week|late-peak|1|**0.8**|\n",
-    "|gasCCGT|R1|2020|all-year|all-week|evening|1|**1**|\n",
-    "|windturbine|R1|2020|all-year|all-week|night|1|1|\n",
-    "|windturbine|R1|2020|all-year|all-week|morning|1|1|\n",
-    "|windturbine|R1|2020|all-year|all-week|afternoon|1|1|\n",
-    "|windturbine|R1|2020|all-year|all-week|early-peak|1|1|\n",
-    "|windturbine|R1|2020|all-year|all-week|late-peak|1|1|\n",
-    "|windturbine|R1|2020|all-year|all-week|evening|1|1|\n"
+    "| ProcessName | RegionName | Time | month    | day      | hour       | UtilizationFactor | MinimumServiceFactor |\n",
+    "|-------------|------------|------|----------|----------|------------|-------------------|----------------------|\n",
+    "| Unit        | -          | Year | -        | -        | -          | -                 | -                    |\n",
+    "| gasCCGT     | R1         | 2020 | all-year | all-week | night      | 1                 | **0.2**              |\n",
+    "| gasCCGT     | R1         | 2020 | all-year | all-week | morning    | 1                 | **0.4**              |\n",
+    "| gasCCGT     | R1         | 2020 | all-year | all-week | afternoon  | 1                 | **0.6**              |\n",
+    "| gasCCGT     | R1         | 2020 | all-year | all-week | early-peak | 1                 | **0.4**              |\n",
+    "| gasCCGT     | R1         | 2020 | all-year | all-week | late-peak  | 1                 | **0.8**              |\n",
+    "| gasCCGT     | R1         | 2020 | all-year | all-week | evening    | 1                 | **1**                |\n",
+    "| windturbine | R1         | 2020 | all-year | all-week | night      | 1                 | 0                    |\n",
+    "| windturbine | R1         | 2020 | all-year | all-week | morning    | 1                 | 0                    |\n",
+    "| windturbine | R1         | 2020 | all-year | all-week | afternoon  | 1                 | 0                    |\n",
+    "| windturbine | R1         | 2020 | all-year | all-week | early-peak | 1                 | 0                    |\n",
+    "| windturbine | R1         | 2020 | all-year | all-week | late-peak  | 1                 | 0                    |\n",
+    "| windturbine | R1         | 2020 | all-year | all-week | evening    | 1                 | 0                    |\n"
```

Comment:
This looks a lot uglier on github than it did in pycharm, but it's probably still fine

Reply:
Actually it depends on the screen width